core/src/main/scala/net/ruippeixotog/scalascraper/browser/JsoupBrowser.scala

```diff
@@ -48,6 +48,14 @@ class JsoupBrowser(val userAgent: String = "jsoup/1.8") extends Browser {
     using(inputStream) { _ => JsoupDocument(Jsoup.parse(inputStream, charset, "")) }
 
   def cookies(url: String) = cookieMap.toMap
+ 
+  def setCookie(url: String, key: String, val: String) = {
```

Comment:
The code isn't compiling because `val` is a reserved keyword in Scala. Can you change the name of this parameter to `value`?